src/routes/privateRpc.ts

```diff
@@ -79,6 +79,9 @@ export default function (router: ConnectRouter) {
         ...req,
         id: postId,
         shortId: postId,
+        visible: false,
+        sentAnalyticsReport: false,
```

Comment:
This maybe needs to be true by default. I'm not sure just try to see if there are any side effects. We use it in cdc and cron job i think

Reply:
reverted to db default which is `true`